go.mod

```diff
@@ -1,18 +1,9 @@
-module github.com/go-playground/validator/v10
+module github.com/beautifulentropy/validator/v10
 
 go 1.18
 
 require (
-	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/go-playground/assert/v2 v2.2.0
 	github.com/go-playground/locales v0.14.1
 	github.com/go-playground/universal-translator v0.18.1
```

Comment:
It's surprising to me that this still has dependencies on the locales and universal-translator packages? But I guess that's because this package treats English exactly the same as any other language, and therefore requires these packages to render English error messages.

Reply:
Yeah, I found that to be a little unfortunate as well.